test/unit/presenters/publishing_api/organisation_presenter_test.rb

```diff
@@ -294,7 +294,7 @@ def organisation.summary
     presented_item = present(organisation)
 
     assert_equal("collections", presented_item.content[:rendering_app])
-    assert_equal([{ path: "/courts-tribunals/court-at-mid-wicket", type: "exact" }], presented_item.content[:routes])
+    assert_equal([{ path: "/courts-tribunals/court-at-mid-wicket", type: "prefix" }], presented_item.content[:routes])
```

Comment:
Not an issue for testing this out, but I wonder for later whether we want to leave the courts and tribunals orgs alone since they're not 'ordinary' org pages, and I don't think e.g. have atom feeds, but not going to cause issues for now

Reply:
Good spot, although I don't think it's particularly relevant that they don't have an atom feed. I suppose the question is whether the redirected paths like `/government/organisations/:organisation_id/chiefs-of-staff` *ever* applied to those organisations. I can see that they now live under `/courts-tribunals`, but I wonder if that was always the case...?

Reply:
Addressed in https://github.com/alphagov/whitehall/pull/7621/commits/d4f7bc6299d0fc9eca610b98089e76d01628826a#diff-e861a54893d500d742a133f8c52baa2e9c2a9954262368606f8e5d44ca5924aaR66.